crowbar_framework/app/assets/stylesheets/content/footer.scss

```diff
@@ -33,3 +35,12 @@ footer {
     }
   }
 }
+
+.bottom-line {
+  @include gradient-horizontal(#3f2c50, #502c4f, 0%, 90%);
```

Comment:
Color literals like `#3f2c50` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#502c4f` should only be used in variable declarations; they should be referred to via variable everywhere else.